lib/base/serialize.js

```diff
@@ -57,19 +56,23 @@ module.exports.iso8601DateTime = function(d) {
  * @return {object} flattened map
  */
 module.exports.prefixedCollapsibleMap = function(m, prefix) {
-  if (_.isUndefined(m) || !_.isPlainObject(m)) {
+  if (!m || typeof m !== 'object' || Object.prototype.toString.call(m) !== '[object Object]') {
```

Comment:
I dislike relying on the string representation. Do we need this for our purposes?

Reply:
The string representation is to check that the object being passed in is a "plain object" (since arrays and the like still have the base type of `object`), which is analogous to how lodash already [implements](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L12036) `isPlainObject()`.